tests/func/test_get.py

```diff
@@ -34,46 +32,13 @@ def test_get_repo_file_replace_without_confirmation(tmp_dir, erepo_dir):
 
     Repo.get(os.fspath(erepo_dir), "file", "file_imported")
     # getting another file with a name that already exists in Repo.
-    with pytest.raises(ConfirmRemoveError):
+    with pytest.raises(FileExistsError):
         Repo.get(os.fspath(erepo_dir), "file2", "file_imported")
 
     assert os.path.isfile("file_imported")
     assert (tmp_dir / "file_imported").read_text() == "contents"
 
 
-def test_get_repo_file_replace_yes(tmp_dir, erepo_dir):
```

Comment:
Following our `no-prompt` policy. Plus this didn't work for git files anyway :slightly_frowning_face: